components/src/core/list-item-tag/list-item-tag.component.scss

```diff
@@ -0,0 +1,14 @@
+.pxb-list-item-tag-wrapper {
+    padding: 0;
+    overflow: hidden;
+    
+    line-height: inherit;
+    padding-left: 4px;
+    border-radius: 2px;
+    padding-right: 4px;
+    letter-spacing: 1px;
+}
+
+.pxb-list-item-tag-label {
+    font-weight: bold;
```

Comment:
We don't typically use 'bold' as a value, we use the numerical weights (also we rarely use bold, most often we use semibold):
- 300 (light)
- 400 (regular)
- 600 (semibold)
- 700 (bold)
- 800 (extrabold)